test/e2e/run-test.sh

```diff
@@ -21,13 +21,25 @@ set -x # so users can see what is going on
 env="kind"
 test="bash"
 fail_flag=""
-
+setup_flags=""
+existing_wec_contexts=""
 
 while [ $# != 0 ]; do
     case "$1" in
-        (-h|--help) echo "$0 usage: (--released | --env | --test-type | --kubestellar-controller-manager-verbosity \$num | --transport-controller-verbosity \$num | --fail-fast)*"
+        (-h|--help) echo "$0 usage: (--released | --env \$kind_or_ocp | --test-type \$bash_or_ginkgo | --kubestellar-controller-manager-verbosity \$num | --transport-controller-verbosity \$num | --fail-fast | --existing-wec-contexts \$context1,\$context2,...)*"
```

Comment:
BTW, since this is already using `(` and `|` to organize syntax options, I think that it would be simpler to write `(bash | ginkgo)` than `$bash_or_ginkgo` AND later explain that `$bash_or_ginkgo` means `bash` or `ginkgo`.